unity3d/Assets/Swrve/Editor/SwrveSDKPostProcess.cs

```diff
@@ -10,7 +10,7 @@ namespace AssemblyCSharpEditor
 {
 public class SwrveSDKPostProcess
 {
-    [PostProcessBuild]
+//    [PostProcessBuild]
```

Comment:
I meant to chat to you about this actually, this can now all be done with the new method of registering a `IMPL_APP_CONTROLLER_SUBCLASS`, which we're doing in SwrveUnityAppControllerSub.mm ->
https://github.com/Swrve/swrve-unity-sdk/blob/10780-unity-conversations/unity3d/Assets/Plugins/iOS/SwrveUnityAppControllerSub.mm

Again, I still need to run an iOS build through a few different versions of Unity 5x to 100% this, but we should actually be able to remove this file altogether.